lib/tasks/backfills.rake

```diff
@@ -1,11 +1,17 @@
 desc "These tasks are meant to be run once then removed"
 namespace :backfills do
-  task backfill_conversation_unread_status: :environment do
-    Conversation.where(user_with_unread_messages: nil).includes(:messages).each do |conversation|
-      last_notification = conversation.messages.last.notifications_as_message.last
-      if last_notification&.unread?
-        conversation.update(user_with_unread_messages: last_notification.recipient)
+  task developer_response_rate: :environment do
+    begin
+      ActiveRecord::Migration.check_pending!
+    rescue ActiveRecord::PendingMigrationError => e
+      if e.message.include?("add_response_rate_to_developers")
+        puts e.message
+        exit 1
       end
     end
+    Developer.find_each(batch_size: 50) do |developer|
+      UpdateDeveloperResponseRateJob.perform_now(developer)
+      sleep(0.01)
```

Comment:
Why sleep?

Reply:
Sleep to throttle mass Database insertions between batches of 50.
I'm not a db expert, so I don't totally understand the reasoning or the choice of value, but I've read that this is a GoodIdea™.
Refs:
https://world.hey.com/paulosilva/ruby-on-rails-backfilling-data-136adb66
https://github.com/ankane/strong_migrations#backfilling-data